Project-Descriptions-and-Plans/CVA6/CVA6-plan-approved-2023-2024.md

```diff
@@ -0,0 +1,420 @@
+- Open-source hardware: fpnew (ETH Zürich), first verified in CV32E40Pv2
+- Eclipse Foundation, GitHub
+- Digilent Genesys 2 board
+- JADE Design Automation's Register Manager
+- Simulators: Verilator (open-source), Siemens Questa, Synopsys VCS
+- Synthesis: Vivado (Xilinx), Synopsys Design Compiler (ASIC)
+
+CVA6 can be integrated in the OpenPiton framework to build an SMP multi/many-core CPU.
+
+## List of project outputs
+
+### Documentation:
+
+The document structure has been defined at the PL gate:
+- Requirement specification
+    - Identifies features agreed upon
+    - “What” defined as requirements with identifiers
+    - Can be seen as a datasheet
+- Users’ guide
+    - Includes more details than the specification
+    - For CVA6 integrators and users: HW, SW, ASIC, FPGA… viewpoints
+- Design document
+    - Explains the “How”: design choices…
+    - Not prescriptive, written during or after the design
+    - Useful for next projects and certain certification schemes
+- Verification Environment Specification
+    - User manual for the verification environment testbenches, testcases, verification components, etc.
+    - Description of the testbench structure and theory of operation
+- Design Verification Plans
+    - DVplan, Verification Plan, Vplan: same meaning
+    - Feature-by-feature listing of the Device Under Test
+        - and a description of how it will be verified
+        - and how we know when it is verified (coverage).
+    -   Prepared with the [VPTOOL](https://github.com/openhwgroup/core-v-verif/tree/master/tools/vptool) utility
+
+The specification and users' guide are inputs for design verification plans.
+
+### Design 
+
+- CVA6 configurable RTL source code (TRL5 target)
+- Subsystems and FPGA designs, a.k.a. APU, to exercise CVA6 on development boards...
+
+
+### Verification:
+
+- Versatile generic testbench with adaptation layers for CVA6
+    - Subset compliant with _sustainable open-source solution_ expectactions
+- Test sequences
+- Verification results
+     - Including code coverage and functional coverage
+
+### Software:
+
+- Baremetal BSP for FPGA boards featuring CVA6
+- Linux ports (based on UBoot, OpenSBI and Yocto)
+- Toolchains (compiler...)
+
+LLVM and FreeRTOS are products of related OpenHW projects.
+
+
+## TGs Impacted/Resource requirements
+
+|                 | Staff | Members |
+| :-------------- | :---: | :-----: |
+| Cores TG        |   X   |    X    |
+| Verification TG |   X   |    X    |
+| SW TG           |   X   |    X    |
+
+### OpenHW engineering staff resource plan: requirement and availability
+
+The OpenHW staff are expected to support the project on their scope:
+- Mike Thompson, Director of Engineering, Verification Task Group
+- Florian Zaruba, Director of Engineering, HW & SW Task Groups
+- Davide Schiavone, Director of Engineering, Cores Task Group
+- Duncan Bees, Director, Technical Programs
+
+Florian's experience as the creator of ARIANE is also expected.
+
```

Comment:
"is also required for technical guidance" (or whatever way in which Florian's experience is needed.